include/cloudbreak.bash

```diff
@@ -27,6 +27,7 @@ cloudbreak-conf-tags() {
     env-import DOCKER_TAG_CONSUL 0.5
     env-import DOCKER_TAG_REGISTRATOR v5
     env-import DOCKER_TAG_POSTGRES 9.4.1
+    env-import DOCKER_TAG_POSTFIX latest
```

Comment:
No, please stop forking repositories. A better solution is to refer by image ID.